### ISSUE TITLE
How to train inf-dit

### ISSUE THREAD

**Issue by billfjj** [THUDM/Inf-DiT#28]
I want to fine tune this model, but i am unable to run your training code. Could you please provide a detailed description of the dataset fromat used for your training?

**Comment by ivcylc** [comment]
Have you succeeded in training it?
I am wondering where can i change the batch_size

**Comment by yzy-thu** [comment]
> I want to fine tune this model, but i am unable to run your training code. Could you please provide a detailed description of the dataset fromat used for your training?

webdataset with .jpg

**Comment by yzy-thu** [comment]
> Have you succeeded in training it? I am wondering where can i change the batch_size

scripts/ds_config_zero.json

**Comment by ivcylc** [comment]
> > Have you succeeded in training it? I am wondering where can i change the batch_size
> 
> scripts/ds_config_zero.json

Thanks, and i found the loss when training with bs=2 is large than that with bs=1
Additionally, insterestly, when bs=2, the predicted seems excatedly half of that when bs=1, which cause error

**Comment by billfjj** [comment]
> > I want to fine tune this model, but i am unable to run your training code. Could you please provide a detailed description of the dataset fromat used for your training?
> 
> webdataset with .jpg

I built a tar file with only jpg, but after changing the path of this tar file to the configuration file, the training will report an error that the dataset class does not have the __len__ attribute.

**Comment by BrooksXiaoxi** [comment]
> > > Have you succeeded in training it? I am wondering where can i change the batch_size
> > 
> > 
> > scripts/ds_config_zero.json
> 
> Thanks, and i found the loss when training with bs=2 is large than that with bs=1 Additionally, insterestly, when bs=2, the predicted seems excatedly half of that when bs=1, which cause error
Could you please tell me how you prepared your training data and how you configured the model to train? I am having trouble fine-tuning my own model on my dataset. I would appreciate your response, thanks！！！

**Comment by BrooksXiaoxi** [comment]
> > > Have you succeeded in training it? I am wondering where can i change the batch_size
> > 
> > 
> > scripts/ds_config_zero.json
> 
> Thanks, and i found the loss when training with bs=2 is large than that with bs=1 Additionally, insterestly, when bs=2, the predicted seems excatedly half of that when bs=1, which cause error

Could you please tell me how you prepared your training data and how you configured the model to train? I am having trouble fine-tuning my own model on my dataset. I would appreciate your response, thanks！！！

**Comment by sunrccc** [comment]
> > > > Have you succeeded in training it? I am wondering where can i change the batch_size
> > > 
> > > 
> > > scripts/ds_config_zero.json
> > 
> > 
> > Thanks, and i found the loss when training with bs=2 is large than that with bs=1 Additionally, insterestly, when bs=2, the predicted seems excatedly half of that when bs=1, which cause error
> 
> Could you please tell me how you prepared your training data and how you configured the model to train? I am having trouble fine-tuning my own model on my dataset. I would appreciate your response, thanks！！！

May I ask if you have solved this problem? Could you give me some details?

**Comment by sunrccc** [comment]
> May I ask if you have solved this problem? Could you give me some details?

May I ask if you have solved this problem? Could you give me some details?

**Comment by sunrccc** [comment]
> > I want to fine tune this model, but i am unable to run your training code. Could you please provide a detailed description of the dataset fromat used for your training?
> 
> webdataset with .jpg

Sorry, I don't quite understand your answer. Could you explain it in detail? Thank you very much.
